samples/connector-auth-nodejs/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,44 @@
+﻿
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.7",
+  "version": "1.0.1",
+  "id": "<<CONNECTOR ID>>",
```

Comment:
It is not compulsory that it should be a "CONNECTOR ID".
It can be a random guid, so update the place holder accordingly.
Check in [Manifest Schema Reference](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#:~:text=%20Reference%3A%20Manifest%20schema%20for%20Microsoft%20Teams%20,and%20provides%20pointers%20to%20more%20language...%20More%20) docs: